wolfcrypt/src/fe_x25519_x64.i

```diff
@@ -20,8 +20,6 @@
  */
 
 #if defined(USE_INTEL_SPEEDUP)
-    #define HAVE_INTEL_AVX1
```

Comment:
Why is this define removed?
I can see the need for the rest of the changes but don't understand this one/

Reply:
Look below a fee lines it’s defined again down there. it’s a duplicate